### ISSUE TITLE
Improve annotation formatting

### ISSUE THREAD

**Issue by ashfaqfarooqui** [karlicoss/orger#4]
The heading structure right now ends up looking like a wall of text, not very easy to get an overall context of the highlights.

It would be nice to organize them according to chapters. So the structure would be:
* Book name
** Chapter 1
highlight 1
** Chapter 1
highlight 2
** Chapter 2
highlight 1
...

Plus, the properties drawer can (maybe) have additional information about chapter progress.

**Comment by karlicoss** [comment]
Yeah makes sense. That's more of a kobuddy issue though, since need to extract 
. Also my guess that some books might not have proper chapter structure loaded in Kobo or don't have properly named chapters (e.g. smth like `#(34)text/part0015.html`). But would be interesting to try, going to require changing code somewhere [here](https://github.com/karlicoss/kobuddy/blob/4ba73c2d4cc3cb921ac0801bbf88aa8ec592444c/src/kobuddy/__init__.py#L807) and potentially involves `ContentID`, `Title` fields from `content` table. 
Perhaps I'll try it over the weekend.
